lib/everest.cpp

```diff
@@ -24,7 +26,8 @@ const std::array<std::string, 3> TELEMETRY_RESERVED_KEYS = {{"connector_id"}};
 
 Everest::Everest(std::string module_id_, const Config& config_, bool validate_data_with_schema,
                  const std::string& mqtt_server_address, int mqtt_server_port, const std::string& mqtt_everest_prefix,
-                 const std::string& mqtt_external_prefix, const std::string& telemetry_prefix, bool telemetry_enabled) :
+                 const std::string& mqtt_external_prefix, const std::string& telemetry_prefix, bool telemetry_enabled,
+                 const fs::path& errors_dir) :
```

Comment:
since the errors_dir is already passed to the Config, couldn't we do the generation of the ErrorTypeMap there? So we would not have to change the constructor here